L1Trigger/Phase2L1ParticleFlow/interface/BJetId.h

```diff
@@ -0,0 +1,42 @@
+#ifndef L1TRIGGER_PHASE2L1PARTICLEFLOWS_BJETID_H
+#define L1TRIGGER_PHASE2L1PARTICLEFLOWS_BJETID_H
+
+#include <string>
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
+#include "DataFormats/L1TParticleFlow/interface/PFCandidate.h"
+#include "DataFormats/L1TParticleFlow/interface/PFJet.h"
+
+struct BJetTFCache {
+  BJetTFCache(const std::string &graphPath) : graphDef(tensorflow::loadGraphDef(graphPath)) {}
+  std::unique_ptr<tensorflow::GraphDef> graphDef;
+};
+
+class BJetId {
+public:
+  BJetId(const std::string &iInput,
+         const std::string &iOutput,
+         const BJetTFCache *cache,
+         const std::string &iWeightFile,
+         int iNParticles);
+  ~BJetId();
+
+  void setNNVectorVar();
+  float EvaluateNN();
+  float compute(const l1t::PFJet &iJet, float vz, bool useRawPt);
+
+private:
+  tensorflow::Session *session_;
```

Comment:
Note that Tensorflow `Session` is thread safe, and would be better placed into the `BJetTFCache`. Nowadays the `tensorflow::run()` accepts also `tensorflow::Session const*`.